Classes/Utility/IconUtility.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace Tvp\TemplaVoilaPlus\Utility;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+use TYPO3\CMS\Backend\Utility\BackendUtility;
+use TYPO3\CMS\Core\Imaging\IconRegistry;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+final class IconUtility
```

Comment:
Don't we have this class inside core? Or was this added after v8LTS?

Reply:
the public interface is for getting the ICON not the icon identifier.
Thus it could break between versions and will not be available in 12LTS.